packages/github/src/context.ts

```diff
@@ -40,6 +43,9 @@ export class Context {
     this.eventName = process.env.GITHUB_EVENT_NAME as string
     this.sha = process.env.GITHUB_SHA as string
     this.ref = process.env.GITHUB_REF as string
+    this.refName = process.env.GITHUB_REF_NAME as string
+    this.refType = process.env.GITHUB_REF_TYPE as string
+    this.refProtected = (process.env.GITHUB_REF_PROTECTED as string) === 'true'
```

Comment:
```suggestion
    this.refProtected = new Boolean(process.env.GITHUB_REF_PROTECTED as string)
```

Reply:
The `GITHUB_REF_PROTECTED` is expected to have true, so I think the current implementation is sufficient. We need the opinion of the maintainers about this.

cf. https://docs.github.com/en/actions/learn-github-actions/environment-variables

Reply:
@DenKoren what you think ?

Reply:
I just noticed that it causes a compilation error. We will get as follows

> Type 'Boolean' is not assignable to type 'boolean'.
>  'boolean' is a primitive, but 'Boolean' is a wrapper object. Prefer using 'boolean' when possible.

So the current implementation is good.

Reply:
@peaceiris  yeah I just noticed that couple minutes ago so I added `as boolean` at the end

Reply:
I guess that some users maybe want to control `GITHUB_REF_PROTECTED` with `true/false`. Using `Boolean()` cannot consider that use case. So the following approach (the current implementation) looks good for me.

```ts
this.refProtected = (process.env.GITHUB_REF_PROTECTED as string) === 'true'
```